src/scheduler/data_types.h

```diff
@@ -970,6 +970,7 @@ struct dyn_res
 {
 	char *res;
 	char *program;
+	char *script;
```

Comment:
how is 'script' different from 'program' ?

Reply:
Program consists of arguments which is given directly to popen command. Script just conatins the name of the script. I did this because I did not want scheduler to search for script name every time it executes it.

Reply:
i'd also stick to either 'program' or 'script', using both makes it sound like program is different from script.
How about 'command_line' and 'script_name' ?

Reply:
+1 @agrawalravi90's names